index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="./main.css" />
+    <link rel="stylesheet" href="./index.css" />
```

Comment:
- [ ] Greate job on implementing changes suggested by the previous reviewer, however, your app is still not working yet, this is what actually displaying on your website, `list, add new and contact` nav links are not working, to mean I can not run you project, please fix that and resubmit the work.
<img width="1279" alt="yuu" src="https://user-images.githubusercontent.com/79058364/159552360-0ac244f6-1d24-49a4-a4fa-ab6fb4d83d7d.png">

Reply:
Good day, 
I think you need to install node_modules on your local machine to run luxon.js so I add node modules in .ignore file.
Thank you